lightningd/plugin.c

```diff
@@ -339,6 +339,7 @@ static const char *plugin_response_handle(struct plugin *plugin,
 	 * plugin is parent), so detect that case */
 	if (!was_plugin_destroyed(pd))
 		tal_free(request);
+	return NULL;
```

Comment:
This likely is part of the previous commit.

Reply:
Ack, fixed!